src/app/store/trips/selectors.ts

```diff
@@ -0,0 +1,69 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+import { Trip } from '../../models/trips';
+import { TripState, tripsFeatureKey } from './reducer';
+
+export const selectFeature = createFeatureSelector<TripState>(tripsFeatureKey);
+
+const deriveDatesAndCost = (trip: Trip) => {
```

Comment:
Move this to a utils file in a utils dir and export

Reply:
Will do 👍🏻